src/Views/HelloWorld/index.cshtml

```diff
@@ -1,6 +1,46 @@
+@model IEnumerable<Movie>
+
 @{
     ViewData["Title"] = "Movie List";
 }
 
 <h2>My Movie List</h2>
-<p>Hello from our View Template!</p>
+<button>
+    <a asp-area="" asp-controller="HelloWorld" asp-action="NewMovie" class="navbar-brand">Create Movie</a>
+</button>
+<br><br>
+<h4>This are the movies:</h4>
+<table class="table col-md-12">
+    <thead>
+        <tr>
+            <th>Title</th>
+            <th>Release Date</th>
+            <th>Genre</th>
+            <th>Price</th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Title)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.ReleaseDate)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Genre)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Price)
+            </td>
+            <td>
+                <a asp-action="Edit" asp-route-id="@item.Id">Edit</a> |
+                <a asp-action="Details" asp-route-id="@item.Id">Details</a> |
+                <a asp-action="Delete" asp-route-id="@item.Id">Delete</a>
+            </td>
+        </tr>
+}
+    </tbody>
+</table>
+ 
```

Comment:
add eof

Reply:
Already added, but does not show up in github.

Reply:
dont worry